arc/cli/utils/hybrid_state.py

```
@@ -0,0 +1,154 @@
+        # Create configuration if not exists
+        config_name = Path(result.config_path).stem
+        config_id = await db_client.create_configuration(
+            name=config_name,
+            user_id=self._user_id,
+            initial_config={"path": result.config_path}  # Simplified for now
+        )
+        
+        # Create simulation record
+        simulation_id = await db_client.create_simulation(
+            config_version_id=config_id,  # Using config_id directly for now
+            scenario_set=[s.get("scenario_id", f"scenario_{i}") 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)